src/resources/user/user.service.ts

```diff
@@ -177,6 +177,18 @@ export const createTeam = async (
 ): Promise<TeamDocument> => {
 	logger.debug(`Create new team: %o`, teamData)
 
+	const createdTeams = await TeamModel.find({creator: teamData.creator})
+		.select('name')
+		.exec()
+	if (createdTeams.some(team => team.name === teamData.name)) {
```

Comment:
could use includes here

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
How would you do it here since createdTeams is an array of objects? 🤔